src/Core/Migration/V6_7/Migration1752219159AddLanguageActive.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class Migration1752219159AddLanguageActive extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1752219159;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $addedColumn = $this->addColumn(
+            $connection,
+            table: 'language',
+            column: 'active',
+            type: 'TINYINT(1)',
+            nullable: false,
+            default: '0',
+        );
+
+        if ($addedColumn) {
+            $connection->executeStatement(
+                <<<'SQL'
+                    UPDATE `language`
+                    SET `active` = 1
+                    WHERE 1;
```

Comment:
this migration will fail, if the safe mode is active https://dev.mysql.com/doc/refman/8.4/en/mysql-command-options.html#option_mysql_safe-updates

can we use another where clause?

Reply:
Do you have suggestion what we can do? Would 1=1 do? OR something like `active` = 0?

Reply:
Patched it